operational_analysis/methods/plant_analysis.py

```diff
@@ -75,9 +75,19 @@ def __init__(self, plant, uncertainty_meter=0.005, uncertainty_losses=0.05,
         
         self.num_days_lt= (31, 28.25, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31)
 
-        # Run preprocessing step
+        # Run preprocessing step                                                                                                                                                                                   
         self.calculate_monthly_dataframe()
 
+        # Store start and end of period of record
```

Comment:
Record start and end dates for plant period of record. Record these dates in the reanalysis dataframe